pages/common/pip3.md

```diff
@@ -0,0 +1,32 @@
+# pip3
+
+> Python package manager.
+> More information: <https://pip.pypa.io>.
+
+- Install a package:
+
+`pip3 install {{package_name}}`
+
+- Install a specific version of a package:
+
+`pip3 install {{package_name}}=={{package_version}}`
+
+- Upgrade a package:
+
+`pip3 install -U {{package_name}}`
```

Comment:
Is there a long form of this option, and the ones below? In tldr pages we prefer to use the long-form options where available.

Reply:
Yep I just updated them accordingly in the new pull request